dist/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```

Comment:
Please add a professional README as it's the first point of interaction with the project user.